3.test_cases/23.SMHP-esm2/3.train_fsdp.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+#SBATCH --job-name=esm2-accelerate
+#SBATCH -D .
+#SBATCH --output=accelerate-%x.%j.out
+#SBATCH --nodes=2              # number of nodes
+#SBATCH --ntasks-per-node=1         # number of MP tasks
```

Comment:
Maybe you want 
```suggestion
#SBATCH --exclusive # job has exclusive use of the resource, no sharing
```
instead?